tests/tests.toml

```diff
@@ -13,6 +13,7 @@ debug = true
 auth = true
 auth_type = "xrh"
 maximum_feedback_message_length = 255
+org_overview_limit_hours = 3
```

Comment:
This is intentional, because populating database with mock data for REST API tests uses `CURRENT_TIMESTAMP` but I found out it doesn't take into account the timezone, so it was always -2 hours.... took me longer than it should've to figure that one out :)